public/javascript/change_password.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
```

Comment:
Changing the password doesn't seem to work anymore, I'm not sure why though. One option would be to debug this issue and fix it, or maybe just keep this in the html file for now? Your choice